terraform_alert_config/import_adapter/README.md

```diff
@@ -0,0 +1,68 @@
+# Prepare workspace
+## gcloud
+Information about installation [gcloud installation](https://cloud.google.com/sdk/docs/install)
+## Terraform
+Information about installation [Terraform installation](https://www.terraform.io/downloads.html)
+## Kubernetes
+Information about installation [Kubernetes installation](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
```

Comment:
Let's edit this section a bit (changing the title and formatting)

`
## Installation
The following software must be installed to use this configuration:
1. [gcloud](https://cloud.google.com/sdk/docs/install)
2. [terraform](https://www.terraform.io/downloads.html)
3. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
`